README.md

```diff
@@ -1,17 +1,75 @@
 # Fleet Scheduler
 
-This service is responsible for executing jobs/tasks for Fleet
+Fleet-Scheduler is a collection of tasks that can each be set up as jobs in order to automate many things.
 
-## Usage
+A **task** is a set of functions that are executed in order to make changes to fleet services.
 
+A **job** is a task that is set up to be executed on a schedule.
```

Comment:
It would be worth mentioning that Fleet-Scheduler expects to run through a scheduled cronjob in k8s.